images/base/10-restart.conf

```diff
@@ -0,0 +1,8 @@
+# ensure the service is restarted
+[Service]
+Restart=always
+RestartSec=1
+
+# disable rate limiting
```

Comment:
why disabling the rate limiting?
I experienced problems because a service is restarting continuously but the check wasn´t able to detect it down , can this happen to us?

Reply:
It starts in under a second, the default rate limit of 5 in 10s doesn't seem useful in the context of kind, if we can determine a better one for kind we should do that.

We have no checks for this, instead we're observing if the workloads are up.

kubelet and containerd are our only services.